src/humancellatlas/data/metadata/api.py

```diff
@@ -677,33 +677,74 @@ class Link:
     source_type: str
     destination_id: UUID4
     destination_type: str
+    link_type: str
```

Comment:
Try assigning a default value, s you don't need to specify it in the v5 and vx cases.